packages/application-shell/src/components/version-tracker/version-tracker.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import flowRight from 'lodash/flowRight';
+import { connect } from 'react-redux';
+import { withApplicationContext } from '@commercetools-frontend/application-shell-connectors';
+import pkg from '../../../package.json';
+import * as actions from './actions';
+
+const createVersionMetric = ({ applicationName }) => ({
+  metricName: 'versions',
+  metricLabels: {
+    application: applicationName || 'unknown',
+    package_name: pkg.name,
+    package_version: pkg.version,
+  },
+});
+
+export class VersionTracker extends React.Component {
```

Comment:
Same as the MeasureFirstPaint thing. We can use hooks (e.g. useEffect) as we need to support older versions of React.